commands/bmc.vegman

```diff
@@ -90,6 +95,48 @@ function cmd_datetime {
   subcommand "$@"
 }
 
+# @sudo cmd_datetime_method admin
+# @doc cmd_datetime_method
+# Change time synchronization method
+#  {manual|ntp}
+function cmd_datetime_method {
+  local method="$(busctl get-property ${SETTINGS_SERVICE} \
```

Comment:
We need to think about joining the common code and not copy-pasting it around many files.

Reply:
It is not so easy, actually.
I have already tried, but it either leads to a too complex solution or to very slow handling of commands.

Reply:
You do install anyway. Maybe have a '.common' file and just add it to the end of the 'installed' script for the target?